chemprop/cli/predict.py

```diff
@@ -247,8 +247,8 @@ def main(args):
         trainer = pl.Trainer(
             logger=False,
             enable_progress_bar=True,
-            accelerator="auto",
-            devices=args.n_gpu if torch.cuda.is_available() else 1,
+            accelerator="cpu" if args.gpu_spec == -2 else "auto",
```

Comment:
```suggestion
            accelerator="cpu" if args.gpu_spec == -2 else "gpu",
```
small edge case, but there are many possible accelerators that could be selected sometimes (e.g., MPS can be selected on a GPU-enabled MAC)